### DISCUSSION TITLE
BT-681: Differences between Celex and SDK

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#981] [Q&A] [answered]
We have noticed that, according to CELEX, the BT-681 is not included in contract award notices. In the SDK, however, it is not forbidden in these subtypes. How come?

**Comment by YvesJo** [comment, marked as answer]
Hi,
While BT-681 was not foreseen in Result notices for the 2023 amendment, the value of this Business Term conditions the existence of the other FSR Business Terms (these will not be allowed if BT-681 is set to _false_). To simplify the control, it has been decided to have this BT also in the Result notices and this shall be part of the 2025 amendment.
